### PR TITLE
gtkgui/privatechat.py: implement buddy tab is_important

### PR DESCRIPTION
**Pull request by slook**
+ Added: Consider messages that are explicitly addressed to us by mentioned name, or are from buddies, to be important.

### CODE REVIEW

**Comment by slook** [comment]
> I like to get popup notifications whenever anyone sends me anything.

That is my preference also. Perhaps this is more relevant for Chat Room messages, wherein it might be desirable to be alerted only about buddy chat activity, if that option is enabled, for that might warrant another toggle in preferences.

I have reverted the use of `notify_status`.

File: pynicotine/gtkgui/privatechat.py [hunk 1]
```diff
@@ -398,20 +399,23 @@ def show_notification(self, text):
         self.chats.highlight_user(self.user)
 
         if config.sections["notifications"]["notification_popup_private_message"]:
+            user_type = _("Buddy") if is_buddy else _("User")
```
Comment (mathiascode) [inline pynicotine/gtkgui/privatechat.py:402]: I would prefer two strings, `Private Message from %(user)s` and `Private Message from Buddy %(user)s`

Comment (slook) [reply pynicotine/gtkgui/privatechat.py]: It doesn't seem worth making a new string just for this.

Comment (mathiascode) [reply pynicotine/gtkgui/privatechat.py]: Separate strings would be much clearer for a translator. English is very forgiving when hacking together strings like this, but inserting standalone words into another sentence like this wouldn't work in e.g. one of the other languages I know due to declension.

**Pull request merged**